elyra/pipeline/handlers.py

```diff
@@ -51,8 +56,41 @@ def post(self, *args, **kwargs):
         self.write(json_msg)
         self.flush()
 
-    def __artifact_list_to_str(self, pipeline_array):
-        if not pipeline_array:
-            return "None"
+
+class PipelineSchedulerHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to execute pipelines as batch jobs"""
+
+    @web.authenticated
+    @gen.coroutine
+    def get(self):
+        msg_json = dict(title="Operation not supported.")
+        self.write(msg_json)
+        self.flush()
+
+    @web.authenticated
+    @gen.coroutine
+    def post(self, *args, **kwargs):
+        self.log.debug("Pipeline SchedulerHandler now executing post request")
+
+        pipeline_definition = self.get_json_body()
+
+        self.log.debug("JSON payload: %s", pipeline_definition)
+
+        pipeline = PipelineParser.parse(pipeline_definition)
+
+        if pipeline.export:
+            PipelineProcessorManager.export(pipeline)
+            json_msg = json.dumps({"status": "ok",
+                                   "message": "Pipeline successfully exported"})
         else:
-            return ','.join(pipeline_array)
+            response = PipelineProcessorManager.process(pipeline)
+            json_msg = json.dumps({"status": "ok",
+                                   "message": "Pipeline successfully submitted",
```

Comment:
Per this review comment https://github.com/elyra-ai/elyra/pull/560#discussion_r429986432 I don't think we should include status and dialog text in the response.  Let the UI deal with that (and it doesn't look like the UI even checks `status` or `message` from the response anyway).

Reply:
echoing my response on another comment about this, iirc this is an API "good practice" and should be set whether or not our front end uses it.